docs/tutorials/TracingCode.md

```diff
@@ -195,22 +195,22 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
    @Override
    public CommandResult execute(Model model) throws CommandException {
        ...
-       Person personToEdit = lastShownList.get(index.getZeroBased());
-       Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
-       if (!personToEdit.isSamePerson(editedPerson) && model.hasPerson(editedPerson)) {
+       Person internApplicationToEdit = lastShownList.get(index.getZeroBased());
+       Person editedInternApplication = createEditedPerson(internApplicationToEdit, editPersonDescriptor);
+       if (!internApplicationToEdit.isSamePerson(editedInternApplication) && model.hasPerson(editedInternApplication)) {
            throw new CommandException(MESSAGE_DUPLICATE_PERSON);
        }
-       model.setPerson(personToEdit, editedPerson);
+       model.setPerson(internApplicationToEdit, editedInternApplication);
        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
-       return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedPerson));
+       return new CommandResult(String.format(MESSAGE_EDIT_PERSON_SUCCESS, editedInternApplication));
```

Comment:
Perhaps some of these constants may need some changes as well? I am referring to `MESSAGE_EDIT_PERSON_SUCCESS`?

I saw this in some other parts of the code as well!